### ISSUE TITLE
Fix Failing Review Order Tabbing Test

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4888]
The review order tabbing test is failing and needs to be fixed to pass.
